buildmaster/H1_1JET_319GEV_290PB-1_DIF/metadata.yaml

```diff
@@ -0,0 +1,120 @@
+setname: 'H1_1JET_319GEV_290PB-1_DIF'
+
+nnpdf_metadata:
+  nnpdf31_process: "1JET"
+  experiment: "H1"
+
+arXiv:
+  url: "https://arxiv.org/abs/1611.03421"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1496981"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1496981"
+  version: 1
+
+version: 1
+version_comment: "NA"
+
+implemented_observables:
+
+  - observable: {description: "double differential cross section w.r.t. q^2 and pT", label: '$\frac{d^2\sigma}{dq^{2}dpT}$', units: "$pb/GeV^3$"}
+    observable_name: PTQ2
+    process_type: JET
```

Comment:
I think it is better to use DIS for the time being for these. They are going to share more logic (cuts and so on) with DIS than with JET (which is usually p p -> single jet)

@enocera or maybe we should create a new process type for DIS + Jet? These are the first datasets we implement for this, right?

Reply:
@scarlehoff , @t7phy : my recommendation would be to use two new labels, e.g. DIS_JET and DIS_DIJET. The reason being that the perturbative structure of the partonic cross section shares features both with DIS and with jet/dijet production. So this is relevant when correlating MHOU. Incidentally, note that these DIS+jet data will share with DIS the usual cuts (which are not that relevant, if my recollection is correct), BUT they will also have additional cuts, like pp jets (which are far more relevant). I would include these rules in filters.yaml. Would you like me to do so?

Reply:
Yes, that would be great. We have some logic in validphys to apply DIS cuts even if the process type has extra qualifiers so DIS_JET should work fine.
I'm worried about the kinematic variables however, here Q2 is shared but we might need more or a way of mapping between the old (since the other DIS would be old cd) and new so if you add the new cuts we can start testing where (if) that fails...

Reply:
renamed to DIS_JET